conf/nginx.conf

```diff
@@ -33,7 +33,7 @@ http {
         include /var/lib/hypothesis/nginx_envsubst.conf;
         listen 9083;
         merge_slashes off;
-
+        access_log /dev/stdout;
```

Comment:
What's the difference between putting this in the `server { ... }` block, vs putting it higher up in the `http { ... }` block? It looks like putting it in the `server` block might mean that it applies only to that one virtual server, when the `nginx.conf` might contain multiple `server`'s in the future.

Reply:
Yeah, this should be in the `http { ... }`. See https://www.nginx.com/resources/wiki/start/topics/examples/full/ and https://nginx.org/en/docs/http/ngx_http_log_module.html#access_log. I've pushed a commit to fix it

Reply:
future proofing +1